stdlib/os/__init__.pyi

```diff
@@ -564,14 +564,15 @@ if sys.platform != "win32":
         CLD_KILLED: int
         CLD_STOPPED: int
 
-    # TODO: SCHED_RESET_ON_FORK not available on darwin?
-    # TODO: SCHED_BATCH and SCHED_IDLE are linux only?
     SCHED_OTHER: int  # some flavors of Unix
-    SCHED_BATCH: int  # some flavors of Unix
-    SCHED_IDLE: int  # some flavors of Unix
-    SCHED_SPORADIC: int  # some flavors of Unix
     SCHED_FIFO: int  # some flavors of Unix
     SCHED_RR: int  # some flavors of Unix
+    if sys.platform != "darwin" and sys.platform != "linux":
+        SCHED_SPORADIC: int  # some flavors of Unix
+
+if sys.platform == "linux":
+    SCHED_BATCH: int  # some flavors of Unix
```

Comment:
Does this mean it could exist on non-Linux Unix OSes?

Reply:
I only found it in linux man pages. Sorry, I should have updated these comments